docarray/array/storage/sqlite/seqlike.py

```diff
@@ -48,8 +48,8 @@ def _append(self, doc: 'Document', commit: bool = True, **kwargs) -> None:
         if commit:
             self._commit()
 
-    def __del__(self) -> None:
-        super().__del__()
+    def sync(self) -> None:
+        super().sync()
```

Comment:
also here

Reply:
Oh there is still the if statement after